package.json

```diff
@@ -7,9 +7,10 @@
     "lerna-changelog": "cli.js"
   },
   "scripts": {
-    "build": "babel src -d lib",
+    "build": "npm run clean && babel src --out-dir lib --ignore src/__mocks__/GithubAPI.js,src/__mocks__/Changelog.js",
```

Comment:
We can just ignore in npmignore/files? (I normally but all tests in a test folder not like jest so not sure)